debian/patches/0001-hooks.patch

```diff
@@ -0,0 +1,899 @@
+Description: Remove hooks
```

Comment:
Just because? :smile:

This is also just removing the _example_ hooks, which is an important distinction.  It'd be great if this had enough clues to where we can know when to remove this patch, too.  For example, something like `Remove hooks examples that have failing tests`. :+1: